docker-run.sh

```diff
@@ -34,13 +35,12 @@ then
   npm run start
 elif [[ $NODE_ENV == "development" ]]
 then
-  npm run up
-  npm run seed
+  npm run db_migrate_up
+  npm run db_seed
   npm run develop
 elif [[ $NODE_ENV == "test" ]]
 then
-  npm run up
+  npm run db_migrate_up
+  npm run db_clear
   npm run test
-else
-  echo "Improper NODE_ENV=$NODE_ENV, stopping"
```

Comment:
I have no idea why this is/was breaking.

When I leave this `else` clause in place the shell interpreter completely horks.

```
/home/app/docker-run.sh: line 46: syntax error: unexpected end of file (expecting "fi")
```

Too many `if`/`elif` clauses? Who knows…

Fortunately, when this script runs, `docker compose` has already tried to do some interpolation of `NODE_ENV`.

If you pass nothing, it defaults to `development` (which is valid in this script)
If you pass `test` or `production` you will hit these working `if`/`elif` clauses.

If you pass an unknown value, then the `docker compose` interpolation will fail, because there's no named volume backing the database.

```
$ NODE_ENV=impossible docker compose run members_api
service "members_api_postgres" refers to undefined volume db_data_impossible: invalid compose project
```